packages/typstyle/package.yaml

```diff
@@ -0,0 +1,31 @@
+---
+name: typstyle
+description: |
+  Beautiful and reliable typst code formatter
+homepage: https://github.com/Enter-tainer/typstyle
+licenses:
+  - Apache-2.0
+languages:
+  - Typst
+categories:
+  - Formatter
+
+source:
+  id: pkg:github/Enter-tainer/typstyle@v0.11.28
```

Comment:
hi, i just released v0.11.29 and the file name changed. could you please update this pr to the latest version?

Reply:
completed.